installer/bundle/bundle_skel.sh

```diff
@@ -817,6 +834,21 @@ case "$installMode" in
                     [ $TEMP_STATUS -ne 0 ] && BUNDLE_EXIT_STATUS="$TEMP_STATUS"
                 fi
             done
+            for i in bundles/*-bundle-test.sh; do
+                # If filespec didn't expand, break out of loop
+                [ ! -f $i ] && break
+
+                # It's possible we have a test file without bundle; if so, ignore it
+                BUNDLE=`basename $i -bundle-test.sh`
+                [ ! -f bundles/${BUNDLE}-*universal.*.sh ] && continue
+
+                ./$i
+                if [ $? -eq 0 ]; then
+                    ./bundles/${BUNDLE}-*universal.*.sh --install $FORCE $restartDependencies
```

Comment:
I do not seem to find $restartDependencies  defined anywhere, did you miss it during merge?